azure-pipelines-dash.yml

```diff
@@ -0,0 +1,89 @@
+# C/C++ with GCC
+# Build your C/C++ project with GCC using make.
+# Add steps that publish test results, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/apps/c-cpp/gcc
+
+pr:
+- dash
+
+trigger:
+  batch: true
+  branches:
+    include:
+    - dash
+
+# this part need to be set in UI
+schedules:
+- cron: "0 0 * * 6"
+  displayName: Weekly build
+  branches:
+    include:
+    - dash
+  always: true
+
+variables:
+  - name: BUILD_BRANCH
+    ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+      value: $(System.PullRequest.TargetBranch)
+    ${{ else }}:
+      value: $(Build.SourceBranchName)
+
+stages:
+- stage: Build
+  jobs:
+  - template: .azure-pipelines/build-template.yml
+    parameters:
+      arch: amd64
+      sonic_slave: sonic-slave-buster
+      swss_common_artifact_name: sonic-swss-common
+      swss_common_branch: dash
+      artifact_name: sonic-swss
+      archive_pytests: true
+      archive_gcov: true
+      allow_failed_artifact: true
+      sairedis_artifact_name: sonic-buildimage.vs
+      sairedis_artifact_project: nvidia
+      sairedis_artifact_pipeline: nvidia-sonic.sonic-buildimage
+      sairedis_artifact_branch: bluefield
+      sairedis_artifact_pattern: target/debs/buster
+      common_lib_artifact_name: sonic-buildimage.vs
+      common_lib_artifact_project: nvidia
+      common_lib_artifact_pipeline: nvidia-sonic.sonic-buildimage
+      common_lib_artifact_branch: bluefield
+
+
+- stage: BuildDocker
+  dependsOn: Build
+  condition: succeeded('Build')
+  jobs:
+  - template: .azure-pipelines/build-docker-sonic-vs-template.yml
+    parameters:
+      swss_common_artifact_name: sonic-swss-common
+      swss_common_branch: dash
+      swss_artifact_name: sonic-swss
+      allow_failed_artifact: true
+      artifact_name: docker-sonic-vs
+      buildimage_artifact_name: sonic-buildimage.vs
+      buildimage_artifact_project: nvidia
+      buildimage_artifact_pipeline: nvidia-sonic.sonic-buildimage
+      buildimage_artifact_branch: bluefield
+      sairedis_artifact_name: sonic-buildimage.vs
+      sairedis_artifact_project: nvidia
+      sairedis_artifact_pipeline: nvidia-sonic.sonic-buildimage
+      sairedis_artifact_branch: bluefield
+      sairedis_artifact_pattern: target/debs/buster
+
+- stage: Test
+  dependsOn: BuildDocker
+  condition: succeeded('BuildDocker')
+  jobs:
+  - template: .azure-pipelines/test-docker-sonic-vs-template.yml
+    parameters:
+      log_artifact_name: log
+      gcov_artifact_name: sonic-gcov
+      sonic_slave: sonic-slave-buster
+      swss_common_branch: dash
```

Comment:
Please change to master branch as per https://github.com/sonic-net/sonic-swss/pull/2501#discussion_r1012389884